### DISCUSSION TITLE
css invalid property value when define tokens

### DISCUSSION THREAD

**Discussion by tanangular** [chakra-ui/panda#1017] [Q&A] [answered]
### Description

define "tokens" in panda.config (or config in playground) 

export const config = defineConfig({
  theme: { 
    tokens: {},
    extend: {} ,
  },
}); 

I found css "invalid property key".

### Link to Reproduction

https://play.panda-css.com/1vH08K6zOU

### Steps to reproduce

see in playground.

![1689008126819](https://github.com/chakra-ui/panda/assets/1311673/e15b9037-2110-4d8e-a474-441ac4eb3270)

![1689008207849](https://github.com/chakra-ui/panda/assets/1311673/1b003983-a437-4637-bb1a-87eb71ca9964)

### JS Framework

React

### Panda CSS Version

0.6.0

### Browser

Microsoft Edge Version 114.0.1823.67, Google Chrome 114.0.5735.198

### Operating System

- [X] macOS
- [ ] Windows
- [ ] Linux

**Comment by segunadebayo** [comment, marked as answer]
Hey @tanangular,

You should keep tokens within the `theme.extend` keyword if you intend to extend the default theme tokens and add yours.

Adding the token directly on the `theme` overrides all tokens.

We'll do a better job explaining this in the docs.
